toolkit/scss/_document.scss

```diff
@@ -22,7 +22,11 @@
   @include core-19;
   @include box-sizing(border-box);
   @include inline-block;
-  @extend %question-meta-spacing;
+
+  /* Same as values as `question-meta-spacing` but preserved on mobile */
```

Comment:
The extend brings in some rules that only work on large sizes.  We want them all the time.